app/components/Auth/components/Buttons/SocialAuthButton.js

```diff
@@ -56,7 +56,8 @@ const styles = StyleSheet.create({
         backgroundColor: platform.color.primary,
     }),
     ButtonText: platform => ({
-        flexBasis: 140, // We set this so that it doesnt "cut out long text like spotify"
+        flexBasis: 150, // We set this so that it doesnt "cut out long text like spotify"
```

Comment:
I think for this we should just use a minWidth instead of flexBasis. Seems like thats what the problem was. We will most likely run into this issue with AppleMusic as well haha.

Reply:
Yes, I think that would be better, will fix!